Source/EventFlow/Subscribers/DomainEventPublisher.cs

```diff
@@ -122,15 +167,34 @@ public class DomainEventPublisher : IDomainEventPublisher
         }
 
         private async Task PublishToAsynchronousSubscribersAsync(
```

Comment:
I don't like it :-) 
We don't need to recovery AsyncSubscribers if we failed to schedule them in first place. We need to recovery **AsyncSubscribers scheduling mechanism**
Idea for separate patch — may be PublishToAsynchronousSubscribersAsync could be implemented by `ISubscribeSynchronousToAll` and we could reuse recover mechanism from there.

Reply:
Removed for now